Hunting Queries/SecurityEvent/RareProcbyServiceAccount.txt

```diff
@@ -0,0 +1,61 @@
+// Name: Rare processes run by Service accounts
+// Description: Service accounts normally are supposed to perform a limited set of tasks in a stable environment. The query collects a list of service account and then joins them with rare processes in an environment to detect anomalous behaviours.
+//
+//Id: af02987c-949d-47d5-b0ae-64d8e1b674e2
+//
+//
+// DataSource: #SecurityEvent
+//
+// Techniques: #Persistence, #Discovery, #LateralMovement , #Initial Access, #Execution,
```

Comment:
This should be Tactics - // Tactics: #Persistence, #Discovery, #LateralMovement , #Initial Access, #Execution

Reply:
Done. Thanks.